public/app/features/alerting/__snapshots__/AlertRuleList.test.tsx.snap

```diff
@@ -73,6 +73,12 @@ exports[`Render should render alert rules 1`] = `
       <div
         className="page-action-bar__spacer"
       />
+      <LinkButton
```

Comment:
Maybe this PR is a good opportunity to get rid of this snapshot?

Reply:
Sure!